lua/wikis/commons/Opponent.lua

```diff
@@ -498,7 +498,7 @@ unsuccessful.
 Wikis sometimes provide variants of this function that include wiki specific
 transformations.
 ]]
----@param record table
+---@param record match2opponent|MGIParsedOpponent
 ---@return standardOpponent
 function Opponent.fromMatch2Record(record)
```

Comment:
I would rather have 3 functions

Opponent.fromMatch2Record
Opponent.fromMatchParsedOpponent

These two serve as a mapping layer from their input types into a common function.

This common function could be a new private function, or it could reuse an existing function